docs/supported-services/ecs.rst

```diff
@@ -176,6 +181,12 @@ The `load_balancer` section is defined by the following schema:
       type: <string> # Required. Allowed values: `http`, `https`. 
       timeout: <integer> # Optional. The connection timeout on the load balancer
       https_certificate: <string> # Required if type=https. The ID of the ACM certificate to use on the load balancer.
+      dns_names:
+       - <string> # Optional.
+
+DNS Names
```

Comment:
When I tried to build the docs I got an error about this DNS Names header, and it didn't get rendered. Like in Beanstalk's docs, I actually prefer it without the DNS Names header, so my vote would be remove it if you're ok with that.